lib/process-services/task-list/components/task-details.component.ts

```diff
@@ -173,6 +173,8 @@ export class TaskDetailsComponent implements OnInit, OnChanges {
     noTaskDetailsTemplateComponent: TemplateRef<any>;
 
     showAssignee: boolean = false;
+    showAttachForm: boolean = false;
```

Comment:
the task stand alone component should be responsible to show/hide the attach form component